test/index.js

```diff
@@ -1,11 +1,23 @@
 var assert = require("assert");
 var sinon  = require("sinon");
-var size   = require("lodash.size");
 
 var fastdom = require("fastdom");
 var raf     = require("./raf");
 var Fastdom = require("../");
 
+function size(obj) {
```

Comment:
I'm happy removing `lodash.foreach` in the lib code, but I don't see the need to remove `lodash.size` this in the tests.